### DISCUSSION TITLE
Dashboard with widgets

### DISCUSSION THREAD

**Discussion by fasteddys** [RaythaHQ/raytha#58] [General]
How can we setup a dashboard with widgets

**Comment by apexdodge** [comment]
Transferring to Discussions board

**Comment by apexdodge** [comment]
I created a quick video explaining how to modify the Dashboard code so you can do whatever you want to it. Check it out here: https://youtu.be/SCe9hwccAIU and it should be helpful to anyone else who sees this Discussion thread.

**Comment by fasteddys** [comment]
Here is a nice example for a turnkey one I was referring to, please credit the author @

https://github.com/jdanylko/Tuxboard for turnkey user driven dashboards
